install.sh

```diff
@@ -3,25 +3,25 @@
 set -e
 
 if [ $# -lt 1 ]; then
-    echo $0 {vc.zip sdk.zip target\|target}
+    echo "$0 {vc.zip sdk.zip target|target}"
```

Comment:
This particular line change is made mostly for consistency right - spaces in `$0` shouldn't be an issue here?

Reply:
Yes, that one I made for consistency.

Unimportant tangent: Although if you want to be 100% correct: `echo` *does* accept one flag `-n`. At least GNU echo does. This flag tells echo to not print a newline. So if someone would exec `install.sh` with the name `-n` (this is possible, see `man execve`) it would create unexpected output. But at that point someone is trying already hard just to remove the newline. Why would anyone do that? A solution would be `printf '%s {vc.zip sdk.zip target|target}' "$0"` (but who cares about that).